utils/flare.py

```diff
@@ -169,40 +171,52 @@ def _add_logs_tar(self):
         self._add_log_file_tar(self._dogstatsd_log)
         self._add_log_file_tar(self._jmxfetch_log)
         self._add_log_file_tar(
-            "{0}/*supervisord.log*".format(os.path.dirname(self._collector_log))
+            "{0}/*supervisord.log".format(os.path.dirname(self._collector_log))
```

Comment:
It's not needed, since it's appended here: https://github.com/DataDog/dd-agent/blob/43180b827f48ec359d600eadf96ed1a74daa6838/utils/flare.py#L178